Source/WebKit/Shared/glib/JavaScriptEvaluationResultGLib.cpp

```diff
@@ -121,8 +121,12 @@ static bool isSerializable(GVariant* variant)
         const char* key;
         GVariant* value;
         while (g_variant_iter_loop(&iter, "{&sv}", &key, &value)) {
-            if (!key || !isSerializable(value))
+            if (!key || !isSerializable(value)) {
+                // Normally g_variant_iter_loop() frees this for us, but not if
+                // we break out of the loop and never call it again.
+                g_variant_unref(value);
```

Comment:
I think it would be clearer to use GRefPtr for the GVariant and g_variant_iter_next in this case. Note there's another case in this file. Since we have smart pointers for GVariant maybe it's less risky and confusing to always use g_variant_iter_next in webkit, but I'm ok if we just change the cases where we are leaking.